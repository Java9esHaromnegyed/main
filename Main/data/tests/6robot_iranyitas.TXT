loadArena testArena6.txt
initArena
robotMovement 1 RIGHT
tick 2
robotMovement 1 PUTTY