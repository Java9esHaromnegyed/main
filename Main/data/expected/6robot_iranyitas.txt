arenaLoaded src: testArena6.txt
robotAdded id: 1 name: "player1" pos: [0; 0]
arenaInited
tick
robotMoved id: 1 pos: [0; 2]
tick
robotMoved id: 1 pos: [0; 4]
puttyDropped fromId: 1 pos: [0; 4]
tick
robotMoved id: 1 pos: [3; 4]
robotSlowedDown id: 1 speed: 1.5
puttyDecayed atPos: [3; 4] decay: 1
tick
robotMoved id: 1 pos: [5; 4]